Equinox/Equinox/Menu/MenuItem.swift

```diff
@@ -38,7 +38,7 @@ final class MenuItem: NSMenuItem {
         keyEquivalent charCode: String,
         keyModifier: NSEvent.ModifierFlags,
         action selector: Selector?,
-        target: AnyObject = self as AnyObject,
+        target: AnyObject = MenuItem.self as AnyObject,
```

Comment:
need to ensure menu items that use default target work fine (copy, paste, delete, etc).

Reply:
Reverted, I was a bit too quick in taking the complier warning fix.